djconnectwise/sync.py

```diff
@@ -2215,9 +2225,11 @@ def sync_related(self, instance, sync_config={}):
         self.sync_children(*sync_classes)
 
     def fetch_sync_by_id(self, instance_id, sync_config={}):
-        instance = super().fetch_sync_by_id(instance_id)
+        api_instance = self.get_single(instance_id)
+        instance, result = self.update_or_create_instance(api_instance)
+
         if not instance.closed_flag:
-            self.sync_related(instance, sync_config)
+            self.sync_related(instance, sync_config, result)
```

Comment:
Is the varialbe, `result` is used elsewhere? It seems passed but not returned or used. Or Is the source up to date?

Reply:
Oh, my confusion. Never mind~ Got it.